Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -1,4 +1,5 @@
 using Microsoft.EntityFrameworkCore;
+using OJS.Servers.Ui.Models.Contests;
```

Comment:
This import should not be here. Services should not reference models in Servers. The application cannot build.